stylelint-config.json

```diff
@@ -78,7 +78,9 @@
     "selector-pseudo-class-parentheses-space-inside": "never",
     "selector-pseudo-element-case": "lower",
     "selector-pseudo-element-colon-notation": "double",
-    "selector-pseudo-element-no-unknown": true,
+    "selector-pseudo-class-no-unknown": [true, {
+      "ignorePseudoClasses": ["ng-deep", "label-shown"]
```

Comment:
What is `label-shown`?

Reply:
Dunno why but `:label-shown` in the form-field was failing lint if I added `ng-deep` as an exception